.github/workflows/build.yml

```diff
@@ -242,6 +234,10 @@ jobs:
             xcode: "Xcode_15.2"
           - os: "macos-15"
             arch: "arm"
+          - os: "macos-15"
+            arch: "test"
```

Comment:
This looks weird. Why not have dedicated boolean flag to mark this as build/test only?

Reply:
didn't want to litter the config with too many flags. also 'arch' is a bit of a misnomer (kinda) since its sole purpose is to give the uploaded archive file of the bundle a distinct name. i think it might be better named 'type', since it's the type of the runner. 'arch' sounds i bit more like the resulting build type, which this flag doesn't influence. i don't mind splitting though.

would you rather want mit to split, rename and/or keep it that way?